fn_guardium_insights_integration/fn_guardium_insights_integration/util/data_parsers.py

```diff
@@ -0,0 +1,98 @@
+# -*- coding: utf-8 -*-
+# Copyright © IBM Corporation 2010, 2020
+# (c) Copyright IBM Corp. 2010, 2020. All Rights Reserved.
+# pragma pylint: disable=unused-argument, no-self-use
+import re
+
+
+def map_details_assets(event_data, asset_data):
+    """
+    Based on asset_type needs to map other detail will take some time.
+    """
+    _data_map = {"user_data": [], "consumed": event_data.get("record").get("consumed")}
+    asset_affected = event_data.get("record").get("asset_affected").lower()
+    _data_map["asset_affected"] = asset_affected
+    _event_details = event_data.get("record").get("event_details")
+    for each_details in _event_details:
```

Comment:
can you add more comments as to what you're doing here?

Reply:
ok. I will add comments